src/app/order/components/checkout-page/invoice.component.ts

```diff
@@ -215,6 +227,15 @@ export class InvoiceComponent implements OnInit {
   }
 
 
+  getLeathers() {
+    this.orderService.getLeathers().subscribe((leathers) => {
+      for (const leather of leathers) {
+        this.leatherIdMap[leather.id] = leather;
+      }
+    });
+  }
+
+
```

Comment:
Inside the order, we save only the ID of the leather. By creating a mapping with id and the leather we are able to get the name of the leather to use in invoice.